### PR TITLE
Set and read LTIUser from cookies for dashboards

### PR DESCRIPTION
**Pull request by marcospri**
## Testing

Follow the notes over: https://github.com/hypothesis/lms/pull/6177

### CODE REVIEW

File: lms/security.py [hunk 1]
```diff
@@ -135,6 +135,11 @@ def get_policy(request: Request):
                 partial(get_lti_user_from_bearer_token, location="form")
             )
 
+        if path.startswith("/dashboard/assignment/"):
```
Comment (marcospri) [inline lms/security.py:138]: Read LTIUser from cookies for this endpoint

File: lms/views/dashboard.py [hunk 2]
```diff
@@ -17,10 +18,29 @@ def __init__(self, request) -> None:
         request_method="POST",
     )
     def assignment_redirect_from_launch(self):
+        """
+        Entry point to the dashboards from an LTI launch.
```
Comment (marcospri) [inline lms/views/dashboard.py:22]: Added a comment on these, please double check if they make sense.

File: lms/views/dashboard.py [hunk 3]
```diff
@@ -17,10 +18,29 @@ def __init__(self, request) -> None:
...
+            BearerTokenSchema(self.request).authorization_param(self.request.lti_user)
+            # White space is not allowed as a cookie character, remove the leading part
+            .replace("Bearer ", "")
+        )
+        response.set_cookie(
+            "authorization",
+            value=auth_token,
+            secure=not self.request.registry.settings["dev"],
+            httponly=True,
+            max_age=60 * 60 * 24 * 30,  # 30 Days
```
Comment (marcospri) [inline lms/views/dashboard.py:41]: We'll have to revisit this. Added a note on the project.

Comment (acelaya) [reply lms/views/dashboard.py:41]: Since we have discussed the token expires in one day, should we set the same expiration to the cookie?

**Review by acelaya — APPROVED** [review]
Looks good.

Just posted a comment around the cookie expiration, but there's an in-progress discussion around that, so we can merge as is for now and change that later if needed.

File: lms/views/dashboard.py (hunk 3 above)
Comment (marcospri) [reply lms/views/dashboard.py]: Done :+1:

**Pull request merged**